### ISSUE TITLE
Add accessibility support

### ISSUE THREAD

**Issue by ProgramMax** [ProgramMax/maxGUI#41]
**Is your feature request related to a problem? Please describe.**
maxGUI should be aware of accessibility things such as large fonts, high contrast, low animation, etc. It should expose these to maxGUI users.

**Describe the solution you'd like**
There should be a way to find the end user's preferences (including when they change) and respect them.
